--- libsamplerate-0.1.8/configure.ac.orig	2012-11-06 12:01:53.384692962 +0100
+++ libsamplerate-0.1.8/configure.ac	2012-11-06 12:02:07.544315260 +0100
@@ -16,7 +16,8 @@
 AC_CONFIG_HEADERS([src/config.h])
 
 AM_INIT_AUTOMAKE($PACKAGE_NAME,$PACKAGE_VERSION)
-AM_SILENT_RULES([yes])
+#AM_SILENT_RULES([yes])
+m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 
 dnl Add parameters for aclocal
 AC_SUBST(ACLOCAL_AMFLAGS, "-I M4")
